src/store.js

```diff
@@ -0,0 +1,26 @@
+import { createStore, applyMiddleware } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import thunk from 'redux-thunk';
+import rootReducer from './reducers';
+import { saveState, loadState } from './localStorage';
+
+const initialState = {};
+const middleware = [thunk];
```

Comment:
Preporucujem ti da se upoznas sa alternativom stack-a Redux + Thunk a to je React Context https://reactjs.org/docs/context.html. To je sada aktuelna tehnologija